.drone.yml

```diff
@@ -5000,6 +5000,96 @@ volumes:
 - name: dockersock
   temp: {}
 
+---
+################################################
+# Generated using dronegen, do not edit by hand!
+# Use 'make dronegen' to update.
+# Generated at dronegen/misc.go:35
+################################################
+
+kind: pipeline
+type: kubernetes
+name: publish-apt-new-repos
+trigger:
+  event:
+    include:
+    - push
+  repo: {}
+  branch:
+    include:
+    - rfd/0058-package-distribution
+workspace:
+  path: /go
+clone:
+  disable: true
+steps:
+- name: Check out code
+  image: alpine/git:latest
+  commands:
+  - mkdir -p /go/src/github.com/gravitational/teleport
+  - cd /go/src/github.com/gravitational/teleport
+  - git clone https://github.com/gravitational/${DRONE_REPO_NAME}.git .
+  - git checkout ${DRONE_COMMIT}
+  - mkdir -m 0700 /root/.ssh && echo -n "$GITHUB_PRIVATE_KEY" > /root/.ssh/id_rsa
+    && chmod 600 /root/.ssh/id_rsa
+  - ssh-keyscan -H github.com > /root/.ssh/known_hosts 2>/dev/null && chmod 600 /root/.ssh/known_hosts
+  - rm -f /root/.ssh/id_rsa
+  environment:
+    GITHUB_PRIVATE_KEY:
+      from_secret: GITHUB_PRIVATE_KEY
+- name: Download artifacts
+  image: amazon/aws-cli
+  commands:
+  - mkdir -pv "$ARTIFACT_PATH"
+  - aws s3 sync --no-progress --delete --exclude "*" --include "*.deb*" s3://$AWS_S3_BUCKET/teleport/tag/6.2.31/
+    "$ARTIFACT_PATH"
+  environment:
+    ARTIFACT_PATH: /go/artifacts
+    AWS_ACCESS_KEY_ID:
+      from_secret: AWS_ACCESS_KEY_ID
+    AWS_S3_BUCKET:
+      from_secret: AWS_S3_BUCKET
+    AWS_SECRET_ACCESS_KEY:
+      from_secret: AWS_SECRET_ACCESS_KEY
+- name: Publish debs to APT repos
+  image: golang:1.18.1-bullseye
+  commands:
+  - mkdir -m0700 $GNUPGHOME
+  - echo "$GPG_RPM_SIGNING_ARCHIVE" | base64 -d | tar -xzf - -C $GNUPGHOME
```

Comment:
Does this need post-build cleanup?

Reply:
I can certainly add `rm -rf $GNUPGHOME`, but I figured it was unnecessary as it's stored in a temp in-memory volume with no future steps